mlir/utils/jenkins/miopen-tests/selected-resnet50-miopen-configs

```diff
@@ -0,0 +1,2 @@
+-n 256 -c 1024 -H 14 -W 14 -k 256 -y 1 -x 1 -p 0 -q 0 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -t 1
```

Comment:
Irrelevant to this file: Why the rename from resnext101 -> resnet101? I think you should revert the rename.